### PR TITLE
Fix feature gates flag in kubelet reference

### PR DESCRIPTION
**Pull request by tengqm**
The kubelet reference has to be manually generated and maintained. This PR fixes the outdated list for feature gates, and also  an incorrect default value for WindowsHostNetwork.

### CODE REVIEW

**Comment by kbhawkey** [comment]
@tengqm, can you push a commit to generate a clean preview? Thanks

**Comment by tengqm** [comment]
> @tengqm, can you push a commit to generate a clean preview? Thanks

Trying.

**Comment by dipesh-rawat** [comment]
> Trying.

Moving this PR forward would be advantageous as it holds value.
@tengqm It might be worth considering rebasing this PR against the main (again) and check if that resolves the build errors.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ For a reference to old feature gates that are removed, please refer to
 | `WinDSR` | `false` | Alpha | 1.14 | |
 | `WinOverlay` | `false` | Alpha | 1.14 | 1.19 |
 | `WinOverlay` | `true` | Beta | 1.20 | |
-| `WindowsHostNetwork` | `false` | Alpha | 1.26| |
+| `WindowsHostNetwork` | `true` | Alpha | 1.26| |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:224]: Can you point to why this detail is right? It's also OK to get a tech review from SIG Network.

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:224]: This is correct. The source code is here:

https://github.com/kubernetes/kubernetes/blob/release-1.27/pkg/features/kube_features.go#L1157

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-40811--kubernetes-io-main-staging.netlify.app/docs/reference/command-line-tools-reference/kubelet/

**Pull request merged**